overrides/scripts/expertmode.zs

```diff
@@ -579,80 +530,7 @@ recipes.addShaped(<metaitem:gcym:parallel_hatch.uv>, [
-    .outputs(<contenttweaker:tierfourandhalfship_stabilized>)
-    .duration(5000).EUt(500000).buildAndRegister();
-
-assembly_line.recipeBuilder()
-    .inputs(<contenttweaker:tierfourship>, <contenttweaker:heartofauniverse>, <draconicevolution:chaos_shard> * 4, <metaitem:field.generator.uv> * 24, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6)
-    .fluidInputs(<liquid:rocket_fuel> * 48000, <liquid:omnium> * 576, <liquid:neutronium> * 576)
-    .outputs(<contenttweaker:tierfourship_stabilized>)
-    .duration(5000).EUt(500000).buildAndRegister();
-
-assembly_line.recipeBuilder()
-    .inputs(<contenttweaker:tierthreeship>, <contenttweaker:heartofauniverse>, <draconicevolution:chaos_shard> * 4, <metaitem:field.generator.uv> * 24, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6)
-    .fluidInputs(<liquid:rocket_fuel> * 48000, <liquid:omnium> * 576, <liquid:neutronium> * 576)
-    .outputs(<contenttweaker:tierthreeship_stabilized>)
-    .duration(5000).EUt(500000).buildAndRegister();
-
-assembly_line.recipeBuilder()
-    .inputs(<contenttweaker:tiertwoship>, <contenttweaker:heartofauniverse>, <draconicevolution:chaos_shard> * 4, <metaitem:field.generator.uv> * 24, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6)
-    .fluidInputs(<liquid:rocket_fuel> * 48000, <liquid:omnium> * 576, <liquid:neutronium> * 576)
-    .outputs(<contenttweaker:tiertwoship_stabilized>)
-    .duration(5000).EUt(500000).buildAndRegister();
-
-assembly_line.recipeBuilder()
-    .inputs(<contenttweaker:tieroneship>, <contenttweaker:heartofauniverse>, <draconicevolution:chaos_shard> * 4, <metaitem:field.generator.uv> * 24, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6, <metaitem:plateDenseIridium> * 6)
-    .fluidInputs(<liquid:rocket_fuel> * 48000, <liquid:omnium> * 576, <liquid:neutronium> * 576)
-    .outputs(<contenttweaker:tieroneship_stabilized>)
-    .duration(5000).EUt(500000).buildAndRegister();
-
-// Assembly Control Casing
-recipes.remove(<gregtech:multiblock_casing:3>);
-recipes.addShaped(<gregtech:multiblock_casing:3>, [
-    [<ore:circuitLuv>, <metaitem:plate.high_power_integrated_circuit>, <ore:circuitLuv>],
-    [<metaitem:sensor.iv>, <metaitem:frameTungstenSteel>, <metaitem:emitter.iv>],
-    [<ore:circuitLuv>, <metaitem:electric.motor.iv>, <ore:circuitLuv>]
-]);
-
-// Recycling recipes in groovy
-
-// Assembly Line Casing
-recipes.remove(<gregtech:multiblock_casing:4>);
-recipes.addShaped(<gregtech:multiblock_casing:4>, [
-    [<ore:plateSteel>, <ore:gearRuridit>, <ore:plateSteel>],
-    [<metaitem:robot.arm.iv>, <metaitem:frameTungstenSteel>, <metaitem:robot.arm.iv>],
-    [<ore:plateSteel>, <ore:gearRuridit>, <ore:plateSteel>]
-]);
```

Comment:
Did these get moved to groovy? Can't find them

Reply:
It was made unneded after serenibyss made all casings only make 1.